### DISCUSSION TITLE
Hello, I wanted to install Composer for PhpStorm, but it didn't work out

### DISCUSSION THREAD

**Discussion by Denys-bit** [composer/composer#10761] [General]
Since something is possible with the php file. I wrote php -v to the command line, and it gives errors similar to those when installing Composer
Do not tell me what it could be?

PHP Warning: PHP Startup: Unable to load dynamic library ’curl’ (tried: C:\xampp\php\ext\curl (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_curl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library ’curl’ (tried: C:\xampp\php\ext\curl (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_curl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0
PHP Warning: PHP Startup: Unable to load dynamic library ’openssl’ (tried: C:\xampp\php\ext\openssl (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_openssl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library ’openssl’ (tried: C:\xampp\php\ext\openssl (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_openssl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0
PHP Warning: PHP Startup: Unable to load dynamic library ’php_openssl.dll’ (tried: C:\xampp\php\ext\php_openssl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_php_openssl.dll.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library ’php_openssl.dll’ (tried: C:\xampp\php\ext\php_openssl.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_php_openssl.dll.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0
PHP Warning: PHP Startup: Unable to load dynamic library ’php_ftp.dll’ (tried: C:\xampp\php\ext\php_ftp.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_php_ftp.dll.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library ’php_ftp.dll’ (tried: C:\xampp\php\ext\php_ftp.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.), C:\xampp\php\ext\php_php_ftp.dll.dll (═х эрщфхэ єърчрээ√щ ьюфєы№.)) in Unknown on line 0

**Comment by stof** [comment]
those warnings indicate a broken PHP setup. that's what you need to fix first.

**Reply by Denys-bit** [reply]
Thank you! But as always, after sending the question, I myself understood how to solve the problem, it turned out that the php file was crooked, reinstalling php helped
